dart_test.yaml

```diff
@@ -10,15 +10,23 @@ tags:
 
 presets:
   dartdevc:
```

Comment:
This preset is very similar to `unit` below, and per our other conversation about the `dart2js` preset it sounds like we want to run the entire browser suite on dart2js anyway. On top of that, you can't actually specify the web-compiler (ddc vs dart2js) in `dart_test.yaml`, that has to happen via the build_runner command. So maybe a slightly different approach for presets would work better:

```yaml
# Specify chrome and VM as default platforms for running all tests,
# then let the `@TestOn()` annotations determine which suites actually run
platforms:
  - chrome
  - vm

# Default concurrency of 4 for unit tests, integration preset will override
concurrency: 4

# Default paths to include all unit tests, integration preset will override
paths:
  - test/vm_tests/
  - test/over_react_component_declaration_test.dart
  - test/over_react_component_test.dart
  - test/over_react_dom_test.dart
  - test/over_react_util_test.dart

presets:
  dartdevc:
    exclude_tags: no-ddc
  travis:
    reporter: expanded
  integration:
    paths:
      - test/integration/
```

```
# unit tests (ddc)
$ pub run build_runner test
# unit tests (dart2js)
$ pub run build_runner test -r
# integration tests (ddc)
$ pub run build_runner test -- -P integration
# integration tests (dart2js)
$ pub run build_runner test -r -- -P integration
```

Reply:
oh I like that!

Reply:
We still need a preset for dart2js, since there's no platform differentiator between dart2js/ddc

Reply:
So I ended up having to create presets for both dart2js and ddc. ddc needs to ignore the no-ddc tests, and dart2js can't load vm_tests.